src/ServiceControl/DbMigrations/1.41.3/RerunClassifiersMigration.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Particular.ServiceControl.DbMigrations
+{
+    using global::ServiceControl.Recoverability;
+    using Raven.Client;
+
+    public class RerunClassifiersMigration_1_41_3 : IMigration
+    {
+        public string Apply(IDocumentStore store)
+        {
+            using (var session = store.OpenSession())
+            {
+                var reclassifySettings = session.Load<ReclassifyErrorSettings>(ReclassifyErrorSettings.IdentifierCase);
+
+                if (reclassifySettings != null)
+                {
+                    reclassifySettings.ReclassificationDone = false;
```

Comment:
Verified, you are correct. I've submitted another PR into this one to run the migration synchronously as part of the setup. https://github.com/Particular/ServiceControl/pull/989